internal/services/automation/automation_variable_string_resource_test.go

```diff
@@ -75,6 +75,21 @@ func TestAccAutomationVariableString_update(t *testing.T) {
 	})
 }
 
+func TestAccAutomationVariableString_encrypted(t *testing.T) {
```

Comment:
ditto with https://github.com/hashicorp/terraform-provider-azurerm/pull/14887/files#r786637877

Reply:
@tombuildsstuff I've updated the PR. please bother to check again. Thanks!